compiler/noirc_evaluator/src/ssa/opt/normalize_value_ids.rs

```diff
@@ -76,13 +76,14 @@ impl Context {
             new_function.dfg.make_global(value.get_type().into_owned());
         }
 
-        let mut reachable_blocks = PostOrder::with_function(old_function).into_vec();
-        reachable_blocks.reverse();
-
+        let reachable_blocks = old_function.reachable_blocks().into_iter().collect::<Vec<_>>();
         self.new_ids.populate_blocks(&reachable_blocks, old_function, new_function);
 
+        let mut rpo = PostOrder::with_function(old_function).into_vec();
```

Comment:
Please rename or comment that `rpo` is Reverse Post Order

Reply:
I found the code to be pretty self-documenting but I will rename